show/main.py

```diff
@@ -1227,10 +1227,22 @@ def users(verbose):
 
 @cli.command()
 @click.option('--since', required=False, help="Collect logs and core files since given date")
+@click.option('--force', is_flag=True, help="Enable hardware register dump even with system interruption")
```

Comment:
I wonder if we should choose a different name other than "force" for this. Because it doesn't really force anything; it enables disruptive HW register dump. Any other suggestions?

Reply:
I think it will be better to use more specific option like --stop_pmon or --stop_process. Let me check with Dell what exact action needs to be done for the disruptive hw register dump case.

Reply:
@jleveque I think "--allow_process_stop" might be better for this kind of dump case and since it's obvious for user to know the intention of this flag, we can also remove the "--yes" option from the option check. What do you think?

Reply:
I think `--allow-process-stop` is fine. When you say "remove the `--yes` option", do you mean remove the prompt entirely?

Reply:
correct!

Reply:
Sure. I think that's fine.